app/forms/diversities/disability_detail_form.rb

```diff
@@ -6,72 +6,77 @@ class DisabilityDetailForm
     include ActiveModel::AttributeAssignment
     include ActiveModel::Validations::Callbacks
 
-    attr_accessor :trainee, :disability_ids, :additional_disability
+    attr_accessor :trainee, :disability_ids, :additional_disability, :fields
```

Comment:
Potentially something we can look at in the code review but i think `fields` can be declared with `attr_reader` since we're not allowing it to be set publicly. We may need to do the same for some of the other properties in these forms like `trainee`